### PR TITLE
- issue #101: implement structure for email reports and report setting

### PR DESCRIPTION
**Pull request by pprolancer**

### CODE REVIEW

**Comment by vincentdavis** [comment]
@pedro-walter could you review this and merge if it looks good to you.

**Comment by pedro-walter** [comment]
Will review it tonight!

File: templates/tracking/reports/thankyou.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p>Dear <i>{{ name }}</i>,</p>
+<p>
+    Thanks for your <b>{{ last_month_count }}</b> patient patient_visits in the <b>{{last_month}}</b>. <br/>
```
Comment (pedro-walter) [inline templates/tracking/reports/thankyou.html]: patient_visits should be visits

**Comment by pprolancer** [comment]
@pedro-walter 
I have fixed patient_visits to visits. let me know if another issue.
Thanks

**Pull request merged**